concourse/scripts/tag-release.sh

```diff
@@ -20,17 +20,22 @@ Host github.com
   IdentityFile $(pwd)/git-key
 EOF
 
-version=$(cat release-version/number)
-tag="${TAG_PREFIX}${version}"
+cd paas-cf
+if git tag -l --contains HEAD | grep -E -q 'stage-\d\.\d\.\d'
+then 
+  tag="$(git tag -l --contains HEAD | sed 's/stage/prod/')"
+else
+  version=$(cat ../release-version/number)
+  tag="${TAG_PREFIX}${version}"
+fi
 
 echo Configure Git
-cd paas-cf
 git config --global user.email "${GIT_EMAIL}"
 git config --global user.name "${GIT_USER}"
 git remote add ssh "${GIT_REPO_URL}"
 
 echo "Create tag ${tag}"
-git tag -a "${tag}" -m "Version ${version} passed ${AWS_ACCOUNT} \
+git tag -a "${tag}" -m "Tag ${tag} passed ${AWS_ACCOUNT} \
```

Comment:
I like the tag in the comment more as it is more explicit and you can see whether prod and or stage passed all the tests. And what we are passing is a tag here.